manage/upgrading/version_specific_migration/upgrade_to_python3.rst

```diff
@@ -125,29 +136,26 @@ There are some problems that ``python-modernize`` can not fix on its own.
 It also can make changes that are not really needed.
 You need to closely review all changes after you run this tool.
 
-``python-modernize`` will warn you,
-when it is not sure what to do with a possible problem.
-Check this `Cheat Sheet <http://python-future.org/compatible_idioms.html>`_  with idioms
-for writing Python 2-3 compatible code.
+``python-modernize`` will warn you, when it is not sure what to do with a possible problem.
+Check this `Cheat Sheet <http://python-future.org/compatible_idioms.html>`_  with idioms for writing Python 2/3 compatible code.
 
 ``python-modernize`` adds an import of the compatibility library ``six`` if needed.
-The import is added as the last import,
-therefore it is often necessary to reorder the imports.
-The easiest way is to use ``isort``.
-Check the `Python style guide for Plone <https://docs.plone.org/develop/styleguide/python.html#grouping-and-sorting>`_
-for information about the order of imports and an example configuration for ``isort``.
+The import is added as the last import, therefore it is often necessary to reorder the imports.
+The easiest way is to use `isort <https://pypi.python.org/pypi/isort>`_, which does this for you automatically.
+Check the `Python style guide for Plone <https://docs.plone.org/develop/styleguide/python.html#grouping-and-sorting>`_ for information about the order of imports and an example configuration for ``isort``.
 
+If ``six`` is used in the code, make sure that ``six`` is added to the ``install_requires`` list in the ``setup.py`` of the package.
 
 Installation
 ~~~~~~~~~~~~
 
-Install `modernize <https://pypi.python.org/pypi/modernize>`_ into your Python 3 environment with ``pip``.
+Install ``modernize`` into your Python 3 environment with ``pip``.
 
 .. code-block:: shell
 
     ./bin/pip install modernize
 
-Install `isort <https://pypi.python.org/pypi/isort>`_ into your Python 3 environment with ``pip``.
+Install ``isort`` into your Python 3 environment with ``pip``.
 
 .. code-block:: shell
 
```

Comment:
Why are the links removed, do we link to these packages already somewhere else?
Linking to packages/docs is good practice, not everyone may know or use them already.

Reply:
See line 144, the first time `isort` is mentioned.